### DISCUSSION TITLE
Which of the two facial recognition models, buffalo_l and antelopev2, should I choose?

### DISCUSSION THREAD

**Discussion by nodis** [immich-app/immich#7838] [Q&A] [answered]
### The bug

Which of the two facial recognition models, buffalo_l and antelopev2, should I choose? What is the difference between these two models? Which one is better?

### The OS that Immich Server is running on

Synology

### Version of Immich Server

1.98.1

### Version of Immich Mobile App

1.98.1

### Platform with the issue

- [ ] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
....
```

### Your .env content

```Shell
....
```

### Reproduction steps

```bash
1.
2.
3.
...
```

**Comment by mertalev** [comment, marked as answer]
#5028 has some discussion on this. TL;DR no need to change to antelopev2 unless you feel like experimenting.

**Reply by Sawtaytoes** [reply]
That's not the conclusion I got from that discussion. It was more like "we don't know either way". Is there something you saw that gave you a different impression? I noticed mine was set to `antelopev2`, but not sure why.

**Comment by Rubilmax** [comment]
I personnally have tested many combinations of face detection threshold and facial recognition max distance using both buffalo_l and anelopev2 on a 60k faces database and have come to the conclusion that `buffalo_l` with `0.7` face detection threshold and `0.6` facial recognition max distance and `7` min faces has the highest recall/precision compromise.
In particular, going lower in the face detection threshold identifies faces incorrectly ; and going higher in facial recognition merges too many different faces together.
